### PR TITLE
Us4 form add question

### PR DESCRIPTION
**Pull request by Benjamin-51**
The form that lets a user add a question, and if the form is correct, sends it into the database.

### CODE REVIEW

**Review by MorganeL-pro — APPROVED** [review]
Code works. The user can add a new question with as many answers as he wants.

File: src/Model/QuestionManager.php [hunk 1]
```diff
@@ -11,4 +11,20 @@ public function selectRandomQuestion(): array
...
     }
+
+    public function addQuestion(string $question, int $timeLimit): int
+    {
+        $query = "INSERT INTO " . static::TABLE . " (title, timelimit) VALUES (:question, :timelimit)";
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue(':question', $question, \PDO::PARAM_STR);
+        $statement->bindValue(':timelimit', $timeLimit, \PDO::PARAM_STR);
+        $statement->execute();
+        return $this->selectLastQuestion()["id"];
```
Comment (RomainClair) [inline src/Model/QuestionManager.php:22]: It seems that you want to recreate this : https://www.php.net/manual/fr/pdo.lastinsertid.php
You probably should use it instead

File: src/Controller/ManagementController.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+
+    public function addTimeLimit()
+    {
+        if (isset($_POST["time-limit"]) || gettype($_POST["time-limit"] != "integer" || $_POST["time-limit"] < 0)) {
+            return 45;
+        } else {
+            return $_POST["time-limit"];
+        }
+    }
```
Comment (RomainClair) [inline src/Controller/ManagementController.php:87]: This methods are not controllers so they probably should not be in a Controller class.
You probably could have them in a AddQuestionFormValidator class.

**Review by RomainClair — APPROVED** [review]
The question and answer text validation could be a bit more accurate (size according to database for example)

**Pull request merged**